src/dirac_cwl_proto/job/job_wrapper.py

```diff
@@ -170,8 +170,13 @@ def run_job(self, job: JobSubmissionModel) -> bool:
         self.job_path.mkdir(parents=True, exist_ok=True)
 
         try:
+            jobProcessor = job.jobProcessing.instantiate(self)
```

Comment:
Use `self.runtime_metadata` instead (may be this should be renamed as `self.execution_hooks`) 
```suggestion
```